UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -43,8 +43,7 @@ public static SoundManager Instance
 		}
 	}
 
-	[SerializeField]
-	private string[] RoundEndSounds = new string[]
+	[SerializeField] private string[] RoundEndSounds = new string[]
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6319358)